Client.xcodeproj/project.pbxproj

```diff
@@ -9336,7 +9344,7 @@
 			isa = XCBuildConfiguration;
 			buildSettings = {
 				ALWAYS_EMBED_SWIFT_STANDARD_LIBRARIES = YES;
-				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon_Enterprise;
+				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon_Local;
```

Comment:
What are those app icon changes for?

Reply:
This is an unintentional mistake. Good catch.

Reply:
Fixed at https://github.com/brave/brave-ios/pull/4508/commits/9225c3035d7f4104838a1e6fc5b3730824ddeb8e